Proof-Of-Authority.md

```diff
@@ -0,0 +1,76 @@
+# Proof-Of-Authority (PoA)
+
+## Introduction to PoA (Poof-Of-Authority)
+Unlike PoW (Proof-Of-Work) and PoS (Proof-Of-Stake), replying on miners to mine the block, PoA strategy authorize the specific users to produce blocks with some certain rule. Compared to PoW and PoS, PoA does does not require to consume huge amount of physical energy to generate blocks. 
```

Comment:
Unlike PoW (Proof-Of-Work) and PoS (Proof-Of-Stake), "relying" on miners to mine the block, PoA strategy "authorizes" the specific users to produce blocks with some certain rule. Compared to PoW and PoS, PoA "does does" not require to consume huge amount of physical energy to generate blocks.

Reply:
got it